utils/config/examples/daos_server_mdonssd.yml

```diff
@@ -34,21 +50,20 @@ engines:
     # the first ram class has a combination of the following roles:
     # wal, meta, data
 
-    # The DAOS server assigns the roles to the bdev tiers specified in the
-    # storage description.  If these default assignments are not suitable,
-    # the roles can be assigned explicitly.  To do this, each of the roles
-    # must be assigned to a bdev tier.  Each tier must have one or more roles
-    # assigned.  The wal and meta roles must be assigned to exactly one tier,
+    # MD-on-SSD roles must be assigned explicitly.  To do this, each of the roles
+    # must be assigned to at least one bdev tier and each tier must have one or more
+    # roles assigned.  The wal and meta roles must be assigned to exactly one tier,
```

Comment:
I don't think "The wal and meta roles must be assigned to exactly one tier", the only case we don't support is "wal and data are on same tier but meta is on a dedicated tier".

Reply:
fixed